src/dialects/ibmi/query-generator.js

```diff
@@ -138,20 +142,24 @@ export class IBMiQueryGenerator extends AbstractQueryGenerator {
   }
 
   dropTableQuery(tableName, options) {
-    let table = tableName;
-    let schema;
-
-    if (typeof table === 'object') {
-      schema = table.schema || undefined;
-      table = table.table;
-    } else if (options.schema) {
-      schema = options.schema;
```

Comment:
don't forget to document this in the PR's breaking changes (don't forget CC requires a blank line at the start of the commit body)

Reply:
Good one, added to the description